blocks/article-body-block/chains/article-body/default.test.jsx

```diff
@@ -1471,139 +1480,6 @@ describe('article-body chain', () => {
       expect(wrapper.find('figure').find('figcaption').find('p').length).toEqual(0);
     });
 
-    it('should hide photographer and credit', () => {
```

Comment:
was this removed deliberately?

Reply:
Ahhh good catch 👍  I don't remember why this was removed so I think it was accidental while I was going through and fixing all of the unit tests 🤔  I'll restore this and re-verify locally that the tests are passing

Reply:
Ohhh wait now that I'm re-testing this - I remember why I removed this. It's because this unit test is basically testing implementation details of `ImageMetadata`. It was only working previously because `ImageMetadata` wasn't yet mocked but is now so the test isn't possible to execute the same way. I only realized it when I was changing things to fix all of the tests but it seems like the Article Body block probably shouldn't be testing the exact format that `ImageMetadata` renders with right? Seems like that would be the domain of `ImageMetadata` unit tests to test that logic.